tests/system/data_sources/test_postgres.py

```diff
@@ -60,6 +60,7 @@ def cloud_sql(request):
         # If instance already exists, returns host IP and does not add new data
         postgres_instance.setup()
         postgres_instance.add_data("gs://pso-kokoro-resources/postgres_data.sql")
+        postgres_instance.add_data("gs://pso-kokoro-resources/postgres_data_row.sql")
```

Comment:
Moved the new file postgres_data_row.sql to 
**gs://pso-kokoro-resources/test/postgres/system_integration/**
I also copied gs://pso-kokoro-resources/postgres_data.sql to the above folder but since it might be used in other  development branches, I haven't deleted it for now